exercises/04.unit-test/03.problem.hooks/README.mdx

```diff
@@ -35,7 +35,8 @@ afterEach(async () => {
 is an exception. There are in fact, no tests where I want the real
 `console.error` to be called. So by using a `beforeEach` hook, we can mock the
 `console.error` for every test and restore it after it's finished. Additionally,
-it would be a good idea
+it would be a good idea to have some sort of description of what the additional
+good idea would be 😅
```

Comment:
```suggestion
it would be a good idea to make certain there were no unexpected calls to
`console.error` (we don't want to swallow errors and miss actual problems!).
```